open-media-match/src/OpenMediaMatch/models.py

```diff
@@ -1,15 +1,17 @@
 # Copyright (c) Meta Platforms, Inc. and affiliates.
 
-from . import database as db
-
+from dataclasses import dataclass
+from . import db
 
+@dataclass
 class Bank(db.Model):
     __tablename__ = "banks"
-    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
-    name = db.Column(db.String(255), nullable=False)
-    enabled = db.Column(db.Boolean, nullable=False)
+    id: int = db.Column(db.Integer, primary_key=True, autoincrement=True)
```

Comment:
Wait this works? I'd have expected that typing doesn't complain that int != db.Column

If so, there's an easy way to make this bank class and a neutral interface class have the least amount of duplication.

Reply:
I'm as surprised as you. There's probably some disgusting reflection magic happening somewhere. I'll dig into it a bit more.